### ISSUE TITLE
(vscode) Unnecessary new lines are added indefinitely to docstring for class fields

### ISSUE THREAD

**Issue by 34j** [omnilib/ufmt#114]
I discovered this while using the VSCode Extension. Sorry if this issue has already been resolved.
![Example](https://user-images.githubusercontent.com/55338215/187966674-9b7bb17b-735a-4549-b337-8c07c4627a86.gif)

**Comment by JelleZijlstra** [comment]
I don't remember this bug every being reported to Black, for what it's worth. If you can reproduce it with just Black, please open an issue against Black.

**Comment by 34j** [comment]
I tested on the CLI with the latest ufmt==2.0.0 and black==22.8.0 ,and could not reproduce this issue. 
I have upgraded and built the vscode-ufmt library, but this issue continues to occur, so it could be an issue with either the python api for ufmt or vscode-ufmt.

Code for vscode-ufmt
```python
import ufmt
import ufmt.util

if ufmt.__version__.startswith("1."):
    raise RuntimeError("Requires ufmt >= 2.0.0b1")

document_path = pathlib.Path(document.path).resolve()
source_bytes = document.source.encode("utf-8")
black_config = ufmt.util.make_black_config(document_path)
usort_config = ufmt.types.UsortConfig.find(document_path)
ufmt_result = ufmt.ufmt_bytes(
    document_path,
    source_bytes,
    encoding="utf-8",
    black_config=black_config,
    usort_config=usort_config,
)
result_ = utils.RunResult(ufmt_result.decode("utf-8"), "")
```

I have tried replacing the code with the following and note that it worked.
```
assert document.path
replaced = document.path.replace("\\", '/')
result_ = utils.run_module('ufmt', ['ufmt', 'diff', replaced], use_stdin, cwd, '')
log_to_output(result_.stdout)
log_to_output(result_.stderr)
```
This outputs
'✨ 1 file already formatted ✨\n'

**Comment by amyreese** [comment]
Hmm, I'm still not able to reproduce this issue. That said, I published a new version of the vscode extension that has OS/arch specific builds, and bundles native extensions for Python 3.7-3.10.

I'm not sure if it will fix your issue, but it does include updated versions of black, µfmt, and more. Can you try configuring the extension to use the bundled version of µfmt and see if that helps? 

<img width="517" alt="Screen Shot 2022-09-06 at 12 51 39 AM" src="https://user-images.githubusercontent.com/6444/188578069-608449d5-ab21-4c39-b09b-4e1f4ba8b5c6.png">
<img width="783" alt="Screen Shot 2022-09-06 at 12 51 52 AM" src="https://user-images.githubusercontent.com/6444/188578093-ed28037a-2eb1-4835-a6e0-a05086c4eb4d.png">

**Comment by 34j** [comment]
I have tried both useBundled / fromEnvironment and the issue continues to occur. Oddly enough, when I looked at the output, the issue did not occur. 
(I am not at all familiar with such things, but I have wondered if the type of newlines in the ufmt output is different in docstring and elsewhere and the problem is in the encoding or stdout. I think a quick fix would be to make vscode-ufmt to read and write files through ufmt ( and return `utils.RunResult(None, None)`).)
